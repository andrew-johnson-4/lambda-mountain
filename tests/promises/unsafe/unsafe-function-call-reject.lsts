
import lib2/core/bedrock.lsts;

let :Unsafe f(): Nil = ();

f();
