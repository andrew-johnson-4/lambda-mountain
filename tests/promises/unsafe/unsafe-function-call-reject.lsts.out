Call to unsafe function outside of unsafe block
In File tests/promises/unsafe/unsafe-function-call-reject.lsts Line 6 Column 1
